<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendWidget</class>
 <widget class="QWidget" name="FriendWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>671</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="message_lab">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>消息窗口</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="message_stack_widget">
      <property name="styleSheet">
       <string notr="true">QScrollBar{ background: white; width:5px}
QScrollBar::handle:vertical{ background-color: #86dbcb; min-height: 60px ;width:30px }</string>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QtMaterialAutoComplete" name="message_line">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>点击输入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="message_btn">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {background: rgb(171, 171, 171);} QPushButton {border: none;border-radius:10px;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icon/resource/sendmsg.png</normaloff>:/icon/resource/sendmsg.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>712</x>
     <y>22</y>
     <width>271</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="friend_lab">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>好友列表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="friend_table"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QtMaterialAutoComplete" name="friend_query_line">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>查询用户</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="friend_btn">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {background: rgb(171, 171, 171);} QPushButton {border: none;border-radius:10px;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icon/resource/query.png</normaloff>:/icon/resource/query.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refresh_btn">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {background: rgb(171, 171, 171);} QPushButton {border: none;border-radius:10px;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icon/resource/refresh.png</normaloff>:/icon/resource/refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>235</width>
     <height>90</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="show_message_drawer_btn">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover {background: rgb(171, 171, 171);} QPushButton {border: none;border-radius:10px;}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icon/resource/message.png</normaloff>:/icon/resource/message.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="myself_msgbox_btn">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover {background: rgb(171, 171, 171);} QPushButton {border: none;border-radius:10px;}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icon/resource/myMessageBox.png</normaloff>:/icon/resource/myMessageBox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialAutoComplete</class>
   <extends>QLineEdit</extends>
   <header location="global">qtmaterialautocomplete.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
